frontend/taipy-gui/src/components/Taipy/Input.tsx

```diff
@@ -179,16 +204,22 @@ const Input = (props: TaipyInputProps) => {
                     event.shiftKey,
                     increment
                 );
+
                 if (min !== undefined && Number(newValue) < min) {
+                    updateValueWithDelay(min);
```

Comment:
I guess you would still have the message "warning on bad setState while rendering the component" if changeDelay == 0 ?

Reply:
it only show once then never show again so I think it would be fine

Reply:
If the user do not set the change_delay = 0 then it will use the 300ms default delay

Reply:
one warning is not good enough ;-)